docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -80,6 +80,36 @@ The build will take a few minutes. Once it completes, Airbyte compiled at curren
 
 In `dev` mode, all data will be persisted in `/tmp/dev_root`.
 
+## Add a connector under development to Airbyte
+
+These instructions explain how to run a version of an Airbyte connector that you are developing on (e.g. has not been released yet).
+
+- First, build the platform images and run Airbyte:
+
+```bash
+SUB_BUILD=PLATFORM ./gradlew build
+VERSION=dev docker-compose up
+```
+
+- Then, build the connector image:
+```
+docker build ./airbyte-integrations/connectors/<connector-name> -t airbyte/<connector-name>:dev
+```
```

Comment:
This is what would be run if you executed the `./gradlew` command to build the connector too.

Is it worth to point out that alternative? `./gradlew airbyte-integrations:connectors:<connector-name>:build`